src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,23 @@
+%animation-settings {
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+  transform-origin: bottom center;
+  animation-name: move;
+}
+
+@keyframes move {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
+
+%hand-position-settings {
```

Comment:
It's recommended to use a more descriptive name for the placeholder, for example: %stopwatch-hand-position